### PR TITLE
Added missing consentID

### PR DESCRIPTION
**Pull request by MaherKSantina**

### CODE REVIEW

**Review by sincychacko-frollo — APPROVED** [review]

File: Sources/Aggregation/Aggregation.swift [hunk 1]
```diff
@@ -694,8 +694,8 @@ public class Aggregation: CachedObjects, ResponseHandler {
         - providerAccountID: ID of the provider account to be updated
         - completion: Optional completion handler with optional error if the request fails
      */
-    public func updateProviderAccount(providerAccountID: Int64, loginForm: ProviderLoginForm, completion: FrolloSDKCompletionHandler? = nil) {
-        let request = APIProviderAccountUpdateRequest(loginForm: loginForm)
+    public func updateProviderAccount(providerAccountID: Int64, loginForm: ProviderLoginForm, consentID: Int64?, completion: FrolloSDKCompletionHandler? = nil) {
```
Comment (nickfrollo) [inline Sources/Aggregation/Aggregation.swift:697]: Is this parameter documented?

**Review by nickfrollo — CHANGES_REQUESTED** [review]
I think a docs update is needed

File: Sources/Aggregation/Aggregation.swift (hunk 1 above)
Comment (MaherKSantina) [reply Sources/Aggregation/Aggregation.swift:697]: Yes on it thanks

**Review by nickfrollo — APPROVED** [review]
Looking good thanks! CircleCI is failing tests though

**Pull request merged**